playwright/tests/apps.spec.ts

```diff
@@ -38,7 +38,7 @@ test("TC: SALEOR_119 User should be able to install and configure app from manif
   );
   await installationPage.installAppButton.click();
 
-  await expect(appsPage.successBanner).toBeVisible({ timeout: INSTALLATION_PENDING_TIMEOUT });
+  await appsPage.expectSuccessBanner();
```

Comment:
Should we remove this custom timeout? Installation of the application takes a while so there was a timeout added for the 50s. For PR we are skipping apps tests so right now we cannot say how this influences tests

Reply:
You are right, thx